### ISSUE TITLE
How can I append tags with --set ?

### ISSUE THREAD

**Issue by back-2-95** [docker/buildx#631]
I have target like:

```
target "fpm-73" {
  ...
  tags = ["repo/php:7.3-fpm", "repo/php:7.3-fpm-latest"]
}
```

I'm getting Minor version outside of `docker-bake.hcl`, e.g. `7.3.27` so I'd like to append a tag to the target tags:

```
docker buildx bake --print --pull --push --set fpm-73.tags=repo/php:7.3.27
```

Now with that ^^ the tags list from hcl file gets overriden totally.

**Comment by back-2-95** [comment]
Got around this with:

```
PHP_MINOR_VERSION=7.3.27 docker buildx bake --print --pull --push
```

And in `docker-bake.hcl`:

```
variable PHP_MINOR_VERSION {}

target "fpm-73" {
  ...
  tags = ["repo/php:7.3-fpm", "repo/php:${PHP_MINOR_VERSION}-fpm-latest", "repo/php:7.3-fpm-latest"]
}
```

**Comment by tonistiigi** [comment]
I don't think this is currently possible. We've discussed things like `--set fpm-73.tags+=repo/php:7.3.27`

**Comment by errordeveloper** [comment]
> We've discussed things like `--set fpm-73.tags+=repo/php:7.3.27`

Personally I find mini-languages in flags a bit over the top, `--set` flag is already complex enough. Using HCL variables seems like a viable solution.

**Comment by back-2-95** [comment]
Yeah, I have been quite happy with HCL variable thing there ^^
It also reminds passing args to Dockerfile.

**Comment by thompson-shaun** [comment]
Related to https://github.com/docker/buildx/issues/901?
